client/campaign_message_configuration/README.md

```diff
@@ -0,0 +1,79 @@
+# Campaign Message Configuration
+
+Store campaign message configurations for their respective campaigns. This allows users to switch between campaigns while keeping each of the message configurations and restoring them when the user returns to the original campaign. New campaigns can either be created with customizable default
+settings or from the existing configuration (see the "Transfer Settings" option).
+
+## Getting Started
+
+These instructions will get you a copy of the plugin up and running on your local machine for development and testing purposes. 
+
+### Prerequisites
+
+What you need to install the software:
+
+```
+Linux or Windows 
+King Phisher Minimum Required Version = 1.10.0 
+```
+
+### Installing 
+
+- [ ] Download plugin 
+- [ ] Move plugin to: <code>/opt/king-phisher-plugins/client/</code>

```

Comment:
For installing, rephrase all these to `Use King Phisher Plugin Manager to install and enable plugin.`